core/unit_test/CMakeLists.txt

```diff
@@ -326,8 +326,6 @@ if(Kokkos_ENABLE_HIP)
   # FIXME_HIP
   LIST(REMOVE_ITEM HIP_SOURCES
     hip/TestHIP_AtomicOperations_complexdouble.cpp
```

Comment:
What's the problem with this test?

Reply:
I haven't really investigate the problem with `complex` numbers yet.